openedx/course-discovery
# feat: add mgmt command to update url slugs

This PR adds a mgmt command to update course active slugs from csv file passed as argument.

Testing Instructions:
- Run the command with a csv file as argument
- Make sure the csv file has the following header: `course_uuid`, `course_url_slug`
- Runt the command with the following command: `./manage.py update_course_active_url_slugs --csv_file=<path_to_csv_file>`
- Make sure the command runs without errors and the course active slugs are updated
- Error Handling must be working as expected
- Command related tests are added in `test_update_course_active_url_slugs.py`.
